frappe/utils/print_utils.py

```diff
@@ -87,6 +87,9 @@ def get_print(
 			if pdf:
 				return pdf
 
+	for hook in frappe.get_hooks("on_print_pdf"):
+		frappe.get_attr(hook)(local.form_dict)
```

Comment:
Same feedback as https://github.com/frappe/erpnext/pull/48706/files#r2217177310 for `frappe.call` usage.

Explicitly pass arguments that are required: doctype, name, doc etc.

Reply:
Changed to frappe.call. But I pass `form_dict` as it have it all, or should be specific to only what I use?

Reply:
Explicit APIs are always better than passing around dictionaries.

Reply:
Even if it's slightly more code in framework, it makes life of other developer easier by not having to guess dict contents.

Reply:
@ankush thanks and done.